ui/components/component-library/text/text.scss

```diff
@@ -33,7 +33,7 @@ $text-variants: (
   color: var(--color-text-default);
   font-family: var(--font-family-sans);
 
-  > strong {
+  strong {
```

Comment:
Fixes issue where only the direct strong tag child would be bold font-weight. That meant if it was wrapped in a `span` which many of the translations are they would not receive the styles properly.